.github/workflows/test-on-pr.yaml

```diff
@@ -40,23 +40,41 @@ jobs:
           echo Install Prometheus on minikube
           cd scripts
           ./prometheus_on_minikube.sh -as
-      - name: Deploy autotune
+      - name: Deploy kruize in experiment mode
         run: |
-          echo Deploy autotune
+          echo Deploy kruize in experiment mode
           cp ./manifests/autotune/autotune-operator-deployment.yaml_template ./manifests/autotune/autotune-operator-deployment.yaml_template.old
           sed -e "s/imagePullPolicy: Always/imagePullPolicy: IfNotPresent/g" ./manifests/autotune/autotune-operator-deployment.yaml_template.old > ./manifests/autotune/autotune-operator-deployment.yaml_template
           echo "***************************************************************"
           cat ./manifests/autotune/autotune-operator-deployment.yaml_template
           echo "***************************************************************"
           ./deploy.sh -c minikube -i autotune_operator:test
+          sleep 20
+          kruize_pod=$(kubectl get pod -n monitoring | grep autotune | cut -d " " -f1)
+          kubectl logs $kruize_pod -n monitoring -c autotune > ${GITHUB_WORKSPACE}/kruize_pod_log.txt 2>&1
+          kubectl describe pod $kruize_pod -n monitoring > ${GITHUB_WORKSPACE}/kruize_describe_pod_log.txt 2>&1
```

Comment:
As mentioned @chandrams, please move this to a script that can be moved to the scripts dir. This appears common across all the github workflows

Reply:
Thanks for the review, I have moved this to a script.